src/action/channel.js

```diff
@@ -160,6 +160,7 @@ class ChannelAction {
         limboBalance: pfcc.limboBalance,
         maturityHeight: pfcc.maturityHeight,
         blocksTilMaturity: pfcc.blocksTilMaturity,
+        timeTilAvailable: formatTimeTilMaturity(pfcc.blocksTilMaturity),
```

Comment:
Rename to `getTimeTilAvailable` ... we're getting the time from the blocks.

Reply:
fixed